nix/nix.go

```diff
@@ -43,11 +45,13 @@ func Exec(path string, command []string) error {
 	return errors.WithStack(cmd.Run())
 }
 
-func PkgInfo(pkg string) (*Info, bool) {
+func PkgInfo(nixpkgsCommit, pkg string) (*Info, bool) {
 	buf := new(bytes.Buffer)
-	attr := fmt.Sprintf("nixpkgs.%s", pkg)
-	cmd := exec.Command("nix-env", "--json", "-qa", "-A", attr)
+	exactPackage := fmt.Sprintf("nixpkgs/%s#%s", nixpkgsCommit, pkg)
+	cmd := exec.Command("nix", "search", "--json", exactPackage)
+	cmd.Args = appendExperimentalFeatures(cmd.Args, "nix-command", "flakes")
```

Comment:
When `nix search` needs to download the nixpkgs repository, it shows some download progress. Are we showing it too? I'm wondering if we need to do `cmd.Stderr = os.Stderr`